modules/seqexec/model/src/main/scala/seqexec/model/SequencesQueue.scala

```diff
@@ -6,22 +6,28 @@ package seqexec.model
 import cats._
 import cats.implicits._
 import gem.Observation
+import monocle.Getter
+import monocle.macros.Lenses
 import seqexec.model.enum.Instrument
 
 /**
   * Represents a queue with different levels of details. E.g. it could be a list of Ids
   * Or a list of fully hydrated SequenceViews
   */
+@Lenses
```

Comment:
Does `@Lenses` generate the lenses automatically?

Reply:
Yes, it makes life easier though WartRemover complains 😄